### DISCUSSION TITLE
Multiple Y Axis support

### DISCUSSION THREAD

**Discussion by tervay** [f5/unovis#74] [Q&A]
Back again with another question. I'm looking to have multiple Y axes on a graph; one left and one right:

```ts
<VisXYContainer data={data}>
  <VisLine
    x={useCallback((d: ChartDataPoint) => d.x, [])}
    y={[
      useCallback((d: ChartDataPoint) => d.value1, []),
      useCallback((d: ChartDataPoint) => d.value2, []),
    ]}
  />
  <VisAxis type="x" label="My X" />
  <VisAxis type="y" label="Y 1" />
  <VisAxis type="y" label="Y 2" position="right" />
  <VisTooltip />
  <VisCrosshair template={someTemplate} />
</VisXYContainer>
```

Unfortunately, it only seems to render the first listed `type="y"` axis. Is there a way to have multiple axes? The plots are intrinsically linked but on wildly different numerical scales.

**Comment by rokotyan** [comment]
@tervay We don't support dual-axis charts by default — in the majority of cases it makes them more difficult to read.

However, if you think you really need a dual-axis chart you can try achieving that by using two  `VisXYContainer`s with `position: absolute;` (one on top of another).
